### PR TITLE
[DOC] Adds new site pages and content under Visualize 

### LINKED ISSUE #2216: [DOC] Area, bar, and line chart subpages
**What do you want to do?**
 
- [ ] Request a change to existing documentation
- [X] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
 New content on Dashboards visualization types: area, bar, and line

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.
#1402

### PR DESCRIPTION
**Pull request by vagimeli**
Signed-off-by: vagimeli <vagimeli@amazon.com>

### Description
Adds new site pages and content under the Visualize section on the Documentation site

This PR resolves the rebase issue with PR #2230; all reviews are completed under that PR. This PR is ready to merge. 

### Issues Resolved
#2216
#2230 

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _dashboards/visualize/area.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+1. Access Dashboards by connecting to [http://localhost:5601](http://localhost:5601) from a browser.
+1. Select **Visualize** from the menu and then select **Create visualization**.
+1. Select **Area** from the window.
+1. Select **opensearch_dashboards_sample_data_flights** in the **New Area/Choose a source** window.
+1. Select the calendar icon and set the time filter to **Last 7 days**.
+1. Select **Update**.
+
+## Add aggregations to the area chart
+
+Continuing with the area chart created in the preceding steps, you'll create an area chart that displays the top five logs for flights delayed for every three hours over the last seven days:
```
Comment (cwillum) [inline _dashboards/visualize/area.md:35]: Just thinking in text. Would it be appropriate to call the area chart a visualization at this stage?
"Continuing with the area chart created in the preceding steps, you'll create a visualization that displays the top five logs ..."

File: _dashboards/visualize/area.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+1. Add a **Metrics** aggregation.
+   1. Under **Metrics**, select the **Aggregation** dropdown and choose **Average** and then select the **Field** dropdown and choose **FlightDelayMin**.
```
Comment (cwillum) [inline _dashboards/visualize/area.md:38]: TERMS.md would have you use "dropdown list". Microsoft Manual of Style recommends avoiding "dropdown" alone as a UI element. I think I've been called out on this before, so adding the suggestion here.

File: _dashboards/visualize/area.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+   1. Under **Metrics**, select **Add** to add another Y-axis aggregation. 
+   1. Select the **Aggregation** dropdown and choose **Max** and then select the **Field** dropdown and choose **FlightDelayMin**.
+1. Add a **Buckets** aggregation.
+   1. Select **Add** and then **X-axis** in the **Add Bucket** window.
```
Comment (cwillum) [inline _dashboards/visualize/area.md:42]: This same instruction for selecting "Add" appears repeatedly in the steps. To create the sense of a predictable sequence of events, in the first instance could you do something like ...
"Select **Add** to open the **Add Bucket** window, and then select **X-axis**."
And then follow that same format in subsequent steps.

**Review by cwillum — APPROVED** [review]
Looks great. With minor comments.

File: _dashboards/visualize/area.md (hunk 2 above)
Comment (vagimeli) [reply _dashboards/visualize/area.md:38]: Good catch. Change made.

File: _dashboards/visualize/area.md (hunk 1 above)
Comment (vagimeli) [reply _dashboards/visualize/area.md:35]: Change made.

File: _dashboards/visualize/area.md (hunk 3 above)
Comment (vagimeli) [reply _dashboards/visualize/area.md:42]: Change made.

**Review by Naarcha-AWS — APPROVED** [review]

**Comment by vagimeli** [comment]
Duplicate. See #2621

**Pull request merged**